### PR TITLE
Changed the color of the player.

### PR DESCRIPTION
**Pull request by ehgoodenough**
### Summary ###

- Changed the color of the player.
- Decreased the number of dudes to just one.

### Screenshot ###

what-1

### CODE REVIEW

File: source/scripts/Dude.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ export default class Dude extends Pixi.Sprite {
         this.anchor.x = 0.5
         this.anchor.y = 0.5
         
-        this.position.x = Math.random() * 320
-        this.position.y = Math.random() * 240
+        this.position.x = 640 / 2
```
Comment (rk29) [inline source/scripts/Dude.js]: Magic numbers. Granted, they were magic before this change too, so it's not a net loss.

File: source/scripts/Dude.js [hunk 2]
```diff
@@ -27,15 +27,19 @@ export default class Dude extends Pixi.Sprite {
         
         if (Keyb.isDown("W")) {
             this.position.y -= this.speed.movement * delta.s
+            this.tint = 0xFF0000
```
Comment (rk29) [inline source/scripts/Dude.js:31]: Multiple changes in a single CR. Please split.

File: source/scripts/Dude.js (hunk 1 above)
Comment (rk29) [reply source/scripts/Dude.js]: Also not reflected in the commit comments.

File: source/scripts/Dude.js (hunk 2 above)
Comment (ehgoodenough) [reply source/scripts/Dude.js:31]: Good catch! Is it okay if I respect this in all CRs from now on?

Comment (rk29) [reply source/scripts/Dude.js:31]: Next time there will be punishments...

**Review by rk29 — APPROVED** [review]
Ship it

**Pull request merged**